doc/guides/tracing/index.rst

```diff
@@ -130,32 +92,97 @@ How to serialize and emit fields as well as handling alignment, can be done
 
 
-How to Activate?
-================
+The CTF top layer is enabled using the configuration option
+:option:`CONFIG_TRACING_CTF` and can be used with the different transport
+backends both in synchronous and asynchronous modes.
+
+
+SEGGER SystemView Support
+=========================
+
+Zephyr provides built-in support for `SEGGER SystemView`_ that can be enabled in
+any application for platforms that have the required hardware support.
+
+The payload and format used with SystemView is custom to the application and
+relies on RTT as a transport. Newer versions of SystemView support other
+transports, for example UART or using snapshot mode (both still not
+supported in Zephyr).
+
+To enable tracing support with `SEGGER SystemView`_ add the configuration option
+:option:`CONFIG_SEGGER_SYSTEMVIEW` to your project configuration file and set
+it to *y*. For example, this can be added to the
+:ref:`synchronization_sample` to visualize fast switching between threads::
+
+    CONFIG_STDOUT_CONSOLE=y
+    # enable to use thread names
+    CONFIG_THREAD_NAME=y
+    CONFIG_SEGGER_SYSTEMVIEW=y
+    CONFIG_USE_SEGGER_RTT=y
+    CONFIG_TRACING=y
+
+
+.. figure:: segger_systemview.png
+    :align: center
+    :alt: SEGGER SystemView
+    :figclass: align-center
+    :width: 80%
+
+.. _SEGGER SystemView: https://www.segger.com/products/development-tools/systemview/
+
+
+CPU Stats
+=========
+
+A special tracing format which provides information about percentage of CPU
+usage based on tracing hooks for threads switching in and out, interrupts enters
+and exits (only distinguishes between idle thread, non idle thread and scheduler).
+
+Enable this format with the :option:`CONFIG_TRACING_CPU_STATS` option.
 
-Make sure :option:`CONFIG_TRACING_CTF` is set to ``y``
 
+Transport Backends
+******************
 
-How to Use?
-===========
+The following backends are currently supported:
 
-The resulting CTF output can be visualized using babeltrace or TraceCompass:
+* UART
+* USB
+* File (Using native posix port)
+* RTT (With SystemView)
 
-- The CTF output file can be specified in native posix using the ``-ctf-path``
-  command line option
+Using Tracing
+*************
 
-- Create a new empty directory and copy into it:
+The sample :zephyr_file:`samples/subsys/tracing` demonstrates tracing with
+different formats and backends.
 
-  - The TSDL file (``subsys/tracing/ctf/tsdl/metadata``)
+To get started, the simplest way is to use the CTF format with the ``native_posix``
+port, build the sample as follows:
 
-  - The CTF output file renaming it to ``channel0_0``
+.. zephyr-app-commands::
+   :tool: all
+   :app: samples/subsys/tracing
+   :board: native_posix
+   :gen-args: -DCONF_FILE=prj_native_posix_ctf.conf
+   :goals: build
 
-- The trace can be opened by pointing TraceCompass or babeltrace to this new
-  directory
+You can then run the resulting binary with the option ``-trace-file`` to generate
+the tracing data::
```

Comment:
We could also clarify to "specify the output file path in the newly created ``data`` directory above"